content/app/development/data/prefill/_index.nb.md

```diff
@@ -176,13 +176,14 @@ public async Task DataCreation(Instance instance, object data)
 }
 ```
 
-Bytt ut _Datamodell_ med navnet på C# klassen som er blitt generert basert på xsd-en som
+Bytt ut _Datamodell_ med navnet på C#-klassen som er blitt generert basert på xsd-en som
 ble lastet opp i Altinn Studio. Dersom du bruker en egnet kodeeditor vil du kunne definere felter
 som skal populeres ved bruk av intellisense.
 
 Vær oppmerksom på at dersom du har komplekse typer i modellen din, må disse instansieres før man kan
```

Comment:
Kanskje heller endre setningen i stedet for å legge til en setning etterpå.  Noe ala. 

"Vær oppmerksom på at dersom du har komplekse typer i modellen din, må det verifiseres at disse er instansiert før man kan.. "